ocs_ci/ocs/platform_nodes.py

```diff
@@ -1642,3 +1642,95 @@ def read_default_config(self, default_config_path):
             default_config_dict = yaml.safe_load(f)
 
         return default_config_dict
+
+
+class AZURENodes(NodesBase):
+    """
+    Azure Nodes class
+    """
+    def __init__(self):
+        super(AZURENodes, self).__init__()
+        self.subscription_id = config.ENV_DATA.get("azure_subscription_id")
+        self.client_id = config.ENV_DATA['azure_client_id']
+        self.secret = config.ENV_DATA['azure_secret']
+        self.tenant = config.ENV_DATA['azure_tenant']
+        self.resource_group = config.ENV_DATA['azure_resource_group']
```

Comment:
I see you need this for listing of cluster vms in the azure helper code. Ok.

But since resource group is not known before installation (it's created by openshift installer during cluster deployment), this I will need to change somehow when I add the deployment code.

Reply:
Yes, the resource group is required for listing vms and other vm management functions.

The resouce_group is getting created in the below format
`<cluster_name>-<random_string>-rg`